### ISSUE TITLE
GetIfEntry bug?

### ISSUE THREAD

**Issue by Eugenij-W** [Skulltrail192/One-Core-Api#43]
bug?

https://github.com/Skulltrail192/One-Core-Api/blob/5a61e5632a88193406438e29ef2936669613890d/wrappers/base/iphlpapi_wrapper/iphlpapi.c#L1179

must be:
`if (GetIfEntry(&row) != NO_ERROR)`

**Comment by Eugenij-W** [comment]
> If the function succeeds, the return value is NO_ERROR.

the version in the repository returns an error if successful...
```c
    if (GetIfEntry(&row)==0) 
		return ERROR_FILE_NOT_FOUND;
```
...and in case of a failure, it continues to work.

> #define NO_ERROR    0L

**Comment by Skulltrail192** [comment]
I was fixed at version 3.0x.
